### ISSUE TITLE
Use an explicit library for assertions

### ISSUE THREAD

**Issue by lorisleiva** [metaplex-foundation/beet#29]
As discussed, this is just a reminded task to explicitly import an assertion library such as `npm install assert` instead of relying on polyfills to make it easier for developers to get started with libraries using beets.

**Comment by olivbau** [comment]
I'm trying to use `@metaplex-foundation/mpl-token-metadata` on browser (with [vitejs](https://vitejs.dev/)) and I get this error.
Do you know a work around ?



![image](https://user-images.githubusercontent.com/11538800/215703126-c7e0f489-03d6-47ec-90e2-cc484a49155e.png)

**Comment by grabbou** [comment]
Yes, please! Please make `assert` a dependency, otherwise, we have to polyfill when using various packages that depend on this one!

**Comment by olivbau** [comment]
@lorisleiva 
Maybe add an `assert` package as peer-dependency ?

**Comment by staccDOTsol** [comment]
Sure, I can help you with that. 

To use an explicit library for assertions, you can follow these steps:

1. Install the `assert` library by running the following command in your terminal:

   ```
   npm install assert
   ```

2. Import the `assert` library in your code by adding the following line at the top of your file:

   ```javascript
   const assert = require('assert');
   ```

3. Use the `assert` library to write your assertions. For example:

   ```javascript
   assert.strictEqual(1 + 1, 2, '1 + 1 should equal 2');
   ```

   This assertion checks that the expression `1 + 1` equals `2`, and if it doesn't, it will throw an error with the message `'1 + 1 should equal 2'`.

Using an explicit library for assertions like `assert` can make your code more reliable and easier to debug.
